src/database/config.js

```diff
@@ -43,4 +43,5 @@ module.exports = {
     max: DB_MAX_POOL,
     acquire: DB_ACQUIRE,
   },
+  logging: false,
```

Comment:
This also disables logs on prod?

Reply:
yes, i think its useless on prod since we dont actually log the params. 
i can see why this would be useful but it cannot be output to raw logs if not hard to debug uh. 
@dcshzj mentioned of making our own by formatted sql statements which to be honest sounds like the most ideal solution, but that can be done as a separate pr uh